utils/sl/check_script.py

```diff
@@ -110,19 +105,16 @@ def write_variables(outfile, prefix, variables):
         for variable in variables:
             if variable.upper() == 'PREREQS':
                 newlines += 4
-                outfile.write("%sfrom prereq_tools import PreReqComponent\n"
-                              % prefix)
-                outfile.write("%sscons_temp_env = DefaultEnvironment()\n"
-                              % prefix)
+                outfile.write("%sfrom prereq_tools import PreReqComponent\n" % prefix)
+                outfile.write("%sscons_temp_env = DefaultEnvironment()\n" % prefix)
                 outfile.write("%sscons_temp_opts = Variables()\n" % prefix)
                 outfile.write("%s%s = PreReqComponent(scons_temp_env, "
                               "scons_temp_opts)\n" % (prefix, variable))
                 variables.remove(variable)
         for variable in variables:
             if "ENV" in variable.upper():
                 newlines += 1
-                outfile.write("%s%s = DefaultEnvironment()\n" % (prefix,
-                                                                 variable))
+                outfile.write("%s%s = DefaultEnvironment()\n" % (prefix, variable))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string